third_party/2and3/click/core.pyi

```diff
@@ -293,10 +293,10 @@ class Group(MultiCommand):
     def add_command(self, cmd: Command, name: Optional[str] = ...):
         ...
 
-    def command(self, *args, **kwargs) -> Callable[[_F], _F]:
+    def command(self, *args, **kwargs) -> Callable[[_F], Command]:
```

Comment:
We shouldn't be using the `_F` typevar in this case.

Reply:
Good call.  Fixed!